extensions/true-fantom/network/network-v001.js

```diff
@@ -0,0 +1,477 @@
+              },
+              CONTENT_TYPE: {
+                type: Scratch.ArgumentType.STRING,
+                menu: 'content_type'
+              },
+              RESPONSE_TYPE: {
+                type: Scratch.ArgumentType.STRING,
+                menu: 'response_type'
+              }
+            }
+          },
+          {
+            opcode: 'patch_block',
+            blockType: Scratch.BlockType.REPORTER,
+            text: 'patch [CONTENT_TYPE] [BODY] to [URL] respond [RESPONSE_TYPE]',
+            arguments: {
+              URL: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'https://httpbin.org/patch'
+              },
+              BODY: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'apple'
+              },
+              CONTENT_TYPE: {
+                type: Scratch.ArgumentType.STRING,
+                menu: 'content_type'
+              },
+              RESPONSE_TYPE: {
+                type: Scratch.ArgumentType.STRING,
+                menu: 'response_type'
+              }
+            }
+          },
+          '---',
+          {
+            opcode: 'open_link_block',
+            blockType: Scratch.BlockType.COMMAND,
+            text: 'open [URL] in new tab',
+            arguments: {
+              URL: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'https://extensions.turbowarp.org'
+              }
+            }
+          },
+          {
+            opcode: 'open_window_block',
+            blockType: Scratch.BlockType.COMMAND,
+            text: 'open [URL] in new window with width: [WIDTH] height: [HEIGHT] left: [LEFT] top: [TOP]',
+            arguments: {
+              URL: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'https://extensions.turbowarp.org'
+              },
+              WIDTH: {
+                type: Scratch.ArgumentType.NUMBER,
+                menu: 'default'
+              },
+              HEIGHT: {
+                type: Scratch.ArgumentType.NUMBER,
+                menu: 'default'
+              },
+              LEFT: {
+                type: Scratch.ArgumentType.NUMBER,
+                menu: 'default'
+              },
+              TOP: {
+                type: Scratch.ArgumentType.NUMBER,
+                menu: 'default'
+              }
+            }
+          },
+          {
+            opcode: 'redirect_link_block',
+            blockType: Scratch.BlockType.COMMAND,
+            text: 'redirect this tab to [URL]',
+            arguments: {
+              URL: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'https://extensions.turbowarp.org'
+              }
+            }
+          }
+        ],
+        
+        menus: {
+          content_type: { 
+            acceptReporters: true,
+            items: [
+              {
+                text: '(1) text',
```

Comment:
You don't *need* to use numbers here, by the way

Reply:
Why? It seems convenient